documentation/source/faq/main.rst

```diff
@@ -0,0 +1,212 @@
+Frequently asked questions
+==========================
+
+Please visit `inexor.org <https://inexor.org>`__ and join our `Discord server <https://discord.com/invite/acUW8k7>`__.
+
+----
+
+- `What is Inexor?`_
+- `Which platforms are supported?`_
+- `What is the current state of the project?`_
+- `How is Inexor organized?`_
+- `How to contact us?`_
+- `How to build?`_
+- `Where to find Inexor's documentation?`_
+- `What is Vulkan API?`_
+- `Why is Vulkan API the future?`_
+- `Can you explain Vulkan API in simple terms?`_
+- `How difficult is development with Vulkan API?`_
+- `Does my graphics card support Vulkan API?`_
+- `Will you support other rendering APIs?`_
+
+----
+
+What is Inexor?
+---------------
+
+.. image:: ./faq/InexorLogo.png
```

Comment:
This causes a problem: On the main page this works fine, but if we go to "FAQ" in the wiki manually, the image can't be found since the relative include path isn't correct. We should either include from an absoulte path starting from root, or include the image from the same directory.

Reply:
```suggestion
.. image:: /faq/InexorLogo.png
```

Reply:
Done.